hal/src/boron/network/sara_ncp_client.cpp

```diff
@@ -84,7 +85,11 @@ inline system_tick_t millis() {
 }
 
 const auto UBLOX_NCP_DEFAULT_SERIAL_BAUDRATE = 115200;
-const auto UBLOX_NCP_RUNTIME_SERIAL_BAUDRATE_U2 = 115200;
+const auto UBLOX_NCP_RUNTIME_SERIAL_BAUDRATE_U2 = 921600;
+const auto UBLOX_NCP_RUNTIME_SERIAL_BAUDRATE_R4 = 460800;
```

Comment:
Currently, the baud rate setting is stored in NVM on the current latest cell modem version `L0.0.00.00.05.08,A.02.04`, so it persists across device reboots. This only happens on the R410, and doesn't apply to U210.
Our code assumes that the baud rate setting is not persistent, we always start by setting the USART2 modem's serial baud rate to `115200`, which in this case will cause the modem to not respond if the modem was previously set to a different baud rate like `460800`. We can potentially use a flash variable to check this, but this is not ideal because it will not be backwards compatible - for example, if a user goes back to 1.4.4, they may no longer be able to correctly talk to the cell radio. Need to check with ublox if we can make this change non-persistent similar to U201.

Reply:
Disabled 460800 baudrate for now until we get a response from ublox. Any additional changes after receiving reply from ublox can be made in a separate PR.